src/sagemaker/jumpstart/curated_hub/accessors/file_generator.py

```diff
@@ -65,6 +66,10 @@ def generate_file_infos_from_model_specs(
     files = []
     for dependency in HubContentDependencyType:
         location: S3ObjectLocation = public_model_data_accessor.get_s3_reference(dependency)
+        # Training dependencies will return as None if training is unsupported
+        if not location or is_gated_bucket(location.bucket):
```

Comment:
why are we checking for gated?

Reply:
We don't want to copy any files over from the gated bucket